### ISSUE TITLE
datacarpentry/shell-genomics

### ISSUE THREAD

**Issue by zkamvar** [carpentries/lesson-transition#42]
tracking issues for <https://github.com/datacarpentry/shell-genomics>/

- [x] `index.md:44 [missing file]: [Instructor notes](./guide/index.html)`
- [x] permalink replacement needed: `episodes/01-introduction.md:65 [uninformative link text]: [here](https://www.datacarpentry.org/cloud-genomics/02-logging-onto-cloud/#logging-onto-a-cloud-instance)`
- [x] fix permalinks to internal episodes e.g. `www.datacarpentry.org/shell-genomics`
- [x] fix `www.datacarpentry` to `datacarpentry`
- [x] fix [rendering error in episode 4](https://github.com/datacarpentry/shell-genomics/blob/ab88e6fa62ff7a96b5055f19fabfb80370630798/_episodes/04-redirection.md?plain=1#L119-L143) (Jekyll really needs to DIAF) (to be fixed in https://github.com/datacarpentry/shell-genomics/pull/338)
- [x] glossary needs definition list anchors
- [x] investigate HTML blocks (these are to do [the toggling](https://fishtree-attempt.github.io/shell-genomics/05-writing-scripts.html#transferring-data-between-your-local-machine-and-the-cloud) that the other genomics lessons love to use so well)
- [x] inspect why [this solution block](https://github.com/datacarpentry/shell-genomics/blob/ab88e6fa62ff7a96b5055f19fabfb80370630798/_episodes/05-writing-scripts.md?plain=1#L115-L126) is not rendering properly: https://fishtree-attempt.github.io/shell-genomics/05-writing-scripts.html#discussion1 (I fixed this by manually adding the tag. I do not know why pegboard was not tagging that block, but I do not have much time to investigate and it's not terribly important, either)

```
118:episodes/05-writing-scripts.md:337 [unknown div] <div id="div_unix" style="display:block" markdown="1">
119:episodes/05-writing-scripts.md:390 [unknown div] <div id="div_win" style="display:block" markdown="1">
124:episodes/01-introduction.md:69 [uninformative link text]: [here](https://www.datacarpentry.org/cloud-genomics/02-logging-onto-cloud/#logging-onto-a-cloud-instance)
```

**Comment by zkamvar** [comment]
Note during testing: this repository is at risk of giving us issues during the part of the transition where we push the main branch due to the size of some of the blobs: 

```
from datacarpentry/shell-genomics to fishtree-attempt/shell-genomics (bots)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  7802  100  7712  100    90   6239     72  0:00:01  0:00:01 --:--:--  6312
Enumerating objects: 1800, done.
Counting objects: 100% (1800/1800), done.
Delta compression using up to 8 threads
Compressing objects: 100% (687/687), done.
Writing objects: 100% (1800/1800), 49.49 MiB | 10.27 MiB/s, done.
Total 1800 (delta 1107), reused 1734 (delta 1082), pack-reused 0
remote: Resolving deltas: 100% (1107/1107), done.
remote: warning: See https://gh.io/lfs for more information.
remote: warning: File data/MiSeq/good-data.txt is 54.56 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: warning: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.
To https://github.com/fishtree-attempt/shell-genomics.git
 * [new branch]      main -> main
branch 'main' set up to track 'origin/main'.
✔ Setting active project to '/home/zhian/Documents/Carpentries/Git/carpentries/lesson-transition/sandpaper/datacarpentry/shell-genomics'
✔ Initializing empty, orphan 'gh-pages' branch in GitHub repo 'fishtree-attempt/shell-genomics'
✔ GitHub Pages is publishing from:
• URL: 'https://fishtree-attempt.github.io/shell-genomics/'
• Branch: 'gh-pages'
• Path: '/'
Browse the repository at https://github.com/fishtree-attempt/shell-genomics
```

**Comment by zkamvar** [comment]
fixed in f671f915ba4407abe7f47183768431ec00dab92a
